Client/Clique/Modules/JoinedEvents/JoinedEventsViewController.swift

```diff
@@ -8,7 +8,72 @@
 import UIKit
 
 class JoinedEventsViewController: UIViewController {
+    
```

Comment:
Based on the current implementation, this screen and the logic are exactly the same as the previous screen.

I would advise reusing the same screen instead creating an identical screen twice. You can inject a property (e.g., an enum) that will define what API call needs to be called to get the events.

Another way of doing it would be to define a query parameter and one API call which will define events that are returned from the backend.